File: src/main/java/akka/basic/BasicAkkaMain.java
package akka.basic;

/**
 * Fetch from:
 * https://developer.lightbend.com/start/?group=akka&project=akka-samples-main-java
 * 
 * runMain akka.basic.BasicAkkaMain
 */
public class BasicAkkaMain {
	public static void main(String[] args) {
		/** TODO check the akka.Main */
		akka.Main.main(new String[] { HelloBasicAkka.class.getName() });
	}
}

File: src/main/scala/akka/basic/BasicAkkaMain.scala
package akka.basic

/**
 * https://developer.lightbend.com/start/?group=akka&project=akka-samples-main-scala
 * 
 * runMain akka.basic.BasicAkkaMainSc
 */
object BasicAkkaMainSc {
  def main(args: Array[String]): Unit = {
    akka.Main.main(Array(classOf[HelloBasicAkkaSc].getName))
  }
}